TS3AudioBot/NLog.config

```diff
@@ -3,25 +3,34 @@
       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
       throwConfigExceptions="true">
 
-	<targets>
-		<!-- Config log to file -->
-		<target layout="${longdate}|${pad:padding=5:inner=${level:uppercase=true}}|${callsite:includeNamespace=false} ${message}${onexception:${newline}${exception:format=tostring}}" name="logfile" xsi:type="File" fileName="ts3audiobot.log" />
-		<!-- Config log to console -->
-		<target xsi:type="ColoredConsole"
+  <targets>
+    <!-- Config log to file -->
+    <target layout="${longdate}|${pad:padding=5:inner=${level:uppercase=true}}|${callsite:includeNamespace=false} ${message}${onexception:${newline}${exception:format=tostring}}" name="logfile" xsi:type="File" fileName="/var/log/ts3ab/bot.log" />
+    <!-- Config log to console -->
+    <target xsi:type="ColoredConsole"
 		        name="console"
 		        encoding="utf-8"
 		        layout="${time}|${pad:padding=5:inner=${level:uppercase=true}} ${message}">
-			<highlight-row condition="level == LogLevel.Info" foregroundColor="Cyan"/>
-			<highlight-row condition="level == LogLevel.Warn" foregroundColor="Yellow"/>
-			<highlight-row condition="level == LogLevel.Error" foregroundColor="Red"/>
-			<highlight-row condition="level == LogLevel.Fatal" foregroundColor="Magenta"/>
-		</target>
-	</targets>
+      <highlight-row condition="level == LogLevel.Info" foregroundColor="Cyan"/>
+      <highlight-row condition="level == LogLevel.Warn" foregroundColor="Yellow"/>
+      <highlight-row condition="level == LogLevel.Error" foregroundColor="Red"/>
+      <highlight-row condition="level == LogLevel.Fatal" foregroundColor="Magenta"/>
+    </target>
+    <target layout="${longdate}|${message}${onexception:${newline}${exception:format=tostring}}" name="serverlogfile" xsi:type="File" fileName="/var/log/ts3ab/server.log" />
+  </targets>
 
-	<rules>
-		<logger name="TS3AudioBot.*" minlevel="Info" writeTo="logfile" />
-		<logger name="TS3AudioBot.*" minlevel="Info" writeTo="console" />
-		<logger name="TS3Client.*" minlevel="Info" writeTo="logfile" />
-		<logger name="TS3Client.*" minlevel="Info" writeTo="console" />
-	</rules>
+  <rules>
+    <logger name="TS3AudioBot.Plugins.*" minlevel="Trace" writeTo="logfile" />
+    <logger name="TS3AudioBot.*" minlevel="Debug" writeTo="logfile" />
+    <logger name="TS3AudioBot.*" minlevel="Info" writeTo="console" />
+    <logger name="TS3Client" minlevel="Info" writeTo="logfile" />
+    <logger name="TS3Client.PacketHandler.Rtt" minlevel="Warn" writeTo="logfile" />
+    <logger name="TS3Client.PacketHandler.Raw" minlevel="Warn" writeTo="logfile" />
+    <logger name="TS3Client.PacketHandler.Raw.Voice" minlevel="Warn" writeTo="logfile" />
+    <logger name="TS3Client.PacketHandler.Timeout" minlevel="Warn" writeTo="logfile" />
```

Comment:
Any explicit reason why you push all these loggers default to `Warn`?

Reply:
Because i only want their warnings.

Reply:
Then don't make a pull request with those changes?